client/extensions/wp-job-manager/components/setup/index.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Steps } from './constants';
+import DocumentHead from 'components/data/document-head';
+import Intro from './intro';
+import Main from 'components/main';
+import Wizard from 'components/wizard';
+import { getSiteSlug } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const SetupWizard = ( {
+	slug,
+	stepName = Steps.INTRO,
+	translate,
+} ) => {
+	const steps = [ Steps.INTRO ];
+	const components = {
+		[ Steps.INTRO ]: <Intro />,
+	};
+	const mainClassName = 'wp-job-manager__setup';
+
+	return (
+		<Main className={ mainClassName }>
+			<DocumentHead title={ translate( 'Setup' ) } />
+			<Wizard
+				basePath={ `/extensions/wp-job-manager/setup/${ slug }` }
+				components={ components }
+				forwardText={ translate( 'Continue' ) }
+				steps={ steps }
+				stepName={ stepName } />
+		</Main>
+	);
+};
+
+const connectComponent = connect(
```

Comment:
Really minor point and not a blocker but I think most folks usually prefer to split out [mapStateToProps](https://github.com/reactjs/react-redux/blob/master/docs/api.md) and `mapDispatchToProps`.

So perhaps:
```jsx
const mapStateToProps = state => {
    return {
        slug: getSelectedSiteSlug( state ),
    };
};

export default connect( mapStateToProps )( localize( SetupWizard ) );
```